.github/workflows/sync-docs.yml

```diff
@@ -13,7 +13,7 @@
 name: Sync publish docs
 on:
   push:
-    branches: [main]
+    branches: [main, '0.[0-9]+.[0-9]+']
```

Comment:
One quick enhancement suggestion here, let's follow a standard yaml pattern with below snippet everywhere in CI files!
```
branches:
      - main
      - '0.[0-9]+.[0-9]+'
```

Reply:
followed a standard yaml pattern